packages/mass-fragmentation/src/__tests__/__snapshots__/fragment.test.js.snap

```diff
@@ -401,11 +401,11 @@ exports[`fragment > CCCCC2CCC1C(CC)CC(=O)CC1C2 - with Hose Codes 1`] = `
     ],
     "fragmentType": "acyclic",
     "hoses": [
-      "eF@Hp\\\\IPUA~b\`xUTxUP",
-      "eM@HzCaJBhOIewzJCaUSaU@",
-      "gC\`@Dkj@xPhIP^pr\\\\lQHGBjcaU@",
-      "gJP@DkZhCaB\`eAxcIrsY~bPNEUGBj@",
-      "gNp@DkVjj@xPhIP^hr\\\\lvFOtRApjhxUP",
+      "eF@Hp\\IPUAiF_ImwzJCaUSaU@",
+      "eM@HzCaJBhMXrolvyw}EApjipj\`",
+      "gC\`@Dkj@xPhIPZHXisxcYrw[~bPNEUGBj@",
+      "gJP@DkZhCaB\`eAjabgI_T[NV{S_zI@xUT\\Jj|HP",
+      "gNp@DkVjj@xPhIPZxXirVk_\\[NV{S]u{~bPNEUGBjobDhQa@",
```

Comment:
Should we mark the change as breaking because of this?

Reply:
Yes you may make a breaking change even if I think those hoses code are used nowhere. But just it case it would be easy to track down what changed.
Hose codes are just 'onion' kind code for the environment of atoms.
Before it was marked as isotope 5 of carbon
![image](https://github.com/cheminfo/mass-tools/assets/1484241/fa8dac1d-2b53-416a-ad2c-39872ec34e1f)

Now we add more information about the atom (stored as query features, reason we have setQueryFeatures and getQueryFeaturesObject in openchemlib-js), if it is aromatic, in a cycle, number of H, etc.
![image](https://github.com/cheminfo/mass-tools/assets/1484241/724b4ba6-6e4f-4693-b8cc-7708248a68a7)